images/win/scripts/Installers/Install-WDK.ps1

```diff
@@ -24,7 +24,7 @@ else
 
 
 # `winsdksetup.exe /features + /quiet` installs all features without showing the GUI
-$sdkExitCode = Install-EXE -Url $winSdkUrl -Name "winsdksetup.exe" -ArgumentList ("/features", "+", "/quiet")
+$sdkExitCode = Install-Binary -Url $winSdkUrl -Name "winsdksetup.exe" -ArgumentList ("/features", "+", "/quiet")
```

Comment:
please, concider creating variable $ArgumentList and use it in both function calls like you do in all other files, especially because arguments are the same for both installations.

Reply:
Done. I also removed exit conditions from WDK provisioner, and return `exit $exitCode` for non-zero exit codes to `Install-Binary` function. Thanks!